geotrek/tourism/migrations/0055_auto_20250312_1650.py

```diff
@@ -0,0 +1,47 @@
+# Generated by Django 4.2.19 on 2025-03-12 16:50
+
+from django.db import migrations
+from django.conf import settings
+
+from geotrek.common.utils.file_infos import is_a_non_svg_image
+
+
+def remove_information_desk_incorrect_photos(apps, schema_editor):
+    """
+        Remove InformationDesk photos that are not images or SVGs after
+        backing them up as attachments
+    """
+    InformationDesk = apps.get_model("tourism", "InformationDesk")
+    Attachment = apps.get_model("common", "Attachment")
+
+    FileType = apps.get_model("common", "FileType")
+    filetype, created = FileType.objects.get_or_create(type="InformationDesk file backup")
```

Comment:
make a get_or_create in first attachment creation to prevent file type creation in case there is no file to backup